meeting-minutes/2024-12-18.md

```diff
@@ -0,0 +1,196 @@
+| Pablo      | Quiroga    | Qualys, Inc.                                                | Voting Member             | NO      |
+| Peter      | Gephardt   | IBM                                                         | Member                    | NO      |
+| Przemyslaw | Roguski    | Red Hat                                                     | Voting Member             | YES     |
+| Shridhar   | Chari      | Cisco Systems                                               | Member                    | NO      |
+| Sonny      | van Lingen | Huawei Technologies Co., Ltd.                               | Voting Member             | YES     |
+| Stefan     | Arntzen    | Huawei Technologies Co., Ltd.                               | Voting Member             | YES     |
+| Stefan     | Hagen      | Individual                                                  | Secretary, taking notes   | YES     |
+| Thomas     | Proell     | Siemens                                                     | Member                    | NO      |
+| Thomas     | Schaffer   | Cisco Systems                                               | Voting Member             | YES     |
+| Thomas     | Schmidt    | Federal Office for Information Security (BSI) Germany       | Voting Member             | YES     |
+| Tobias     | Limmer     | Siemens                                                     | Member                    | NO      |
+
+Quorum was reached (12 voting members present).
+
+## 1.3 Procedures for this meeting (Moderator)
+
+## 1.4 Approval of agenda
+
+* Roll Call  
+* Updates:  
+  * Thomas Schmidt reports from the CSAF workshops in the week of the CSAF Community Days 2024 in Munich, Germany
+* Approval of previous meeting minutes (motions carried out per e-mail motions)  
+* Review of actions
+* Presentation of use cases and scenarios
+* Review of outstanding issues and pull requests marked for TC discussion: https://github.com/oasis-tcs/openeox  
+* Next steps  
+
+Agenda was approved.
+
+## 1.5 Approval of previous minutes (Moderator)
+
+None (motions carried out already per e-mail motions).
+
+## 1.6 Review of action items and resolutions (Secretary Stefan)
+
+* ACTION on all to prepare and propose 15 minute presentations of use cases and scenarios for December and January (2025) meetings
+  * Justin: Przemyslaw and Thomas Schmidt have prepared presentations
+  * Przemyslaw: Prepared one usecase for today
+  * Omar: Has prepared a 2 minute presentation of Cisco Use Cases
+  * Thomas Schmidt: Is also prepared for today
+  * Feng: Will present in January 2025
+  * Przemyslaw: Suggests to not only present use cases during the meeting but also trty to close a few of the growing list of open issues
+
+## 1.7 Identification of TC voting members (Secretary)
+
+- the roster is still out of sync so no statements on member status changes possible during meeting
```

Comment:
I suggest once we have the September minutes and attendance report, that we can adjust member status accordingly